c10/core/DispatchKey.h

```diff
@@ -214,6 +215,7 @@ enum class DispatchKey : uint8_t {
   AutogradXLA,
   AutogradXPU,
   AutogradMLC,
+  AutogradHABANA,
```

Comment:
Do you actually need the autograd key? If you don't explicitly use it, you can just not define it here (and add it later if you ever need it).

Reply:
Yes, we are using it a for a few ops. Hence, I added the autograd key.